backend/src/main/resources/application.properties

```diff
@@ -2,4 +2,17 @@ spring.profiles.active=dev
 
 spring.data.neo4j.uri=bolt://localhost:7687
```

Comment:
Note: We need to make sure we don't expose this port when deploying.

Reply:
yeah you are right : |